### ISSUE TITLE
Demo for Grlc

### ISSUE THREAD

**Issue by c-martinez** [NLeSC/TEAM2018#37]
Might be nice to also have a demo on how https://github.com/CLARIAH/grlc/ is used. 

There is a version for CandyGene which I had prepared together with @arnikz. Perhaps that is a good starting point?

**Comment by romulogoncalves** [comment]
@c-martinez is this proposal still up for Team2018 November Sprint?

**Comment by c-martinez** [comment]
Yeah, it would still be nice (and there is still no demo available).

**Comment by romulogoncalves** [comment]
@c-martinez could you provide a list of people who have worked with Grlc?

**Comment by c-martinez** [comment]
In general? Or at NLESC?

**Comment by romulogoncalves** [comment]
NLeSC

**Comment by c-martinez** [comment]
@arnikz , @MartineDeVos , @bpmweel , and maybe @dafnevk have used grlc at some point. Guys, correct me if  I'm wrong.

**Comment by romulogoncalves** [comment]
@c-martinez  is there project hours for this work? If yes, which one?

**Comment by c-martinez** [comment]
Maybe Mediasuite -- but I think there aren't too many hours left. Could you check with @jiskattema ?

**Comment by arnikz** [comment]
@romulogoncalves @c-martinez we could write the hours on the #48 project

**Comment by romulogoncalves** [comment]
@c-martinez were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by c-martinez** [comment]
We mostly worked on making the pbg-ld data available via an easy-to-deploy (docker-compose) infrastructure consisting of:
 - virtuoso containing the data
 - grlc server providing API to query data 
 - jupyter notebook testing the queries

Still a few small adjustments to do for a final version of the SPARQL queries to be ready.

Anything to add @arnikz
